Gemfile

```diff
@@ -83,3 +83,7 @@ group :test do
 end
 
 gem "reverse_markdown", "~> 2.1"
+
+# Add state machine for forms
+gem "aasm", "~> 5.5"
+gem "after_commit_everywhere", "~> 1.0"
```

Comment:
Why is this needed?

Reply:
The `after_commit_everywhere` gem is needed as mentioned in aasm readme https://github.com/aasm/aasm#activerecord. It is used to enabling persisting the the objects state when you call the event name as method with a bang. If the bang is not used then the object state is changed but is not saved in the DB